src/ramses_tx/typed_dicts.py

```diff
@@ -282,9 +298,9 @@ class FaultLogEntry(TypedDict):  # NOTE: not identical to _0418
 
 
 # These are from 31DA...
-class AirQuality(TypedDict):
-    air_quality: float | None
-    air_quality_basis: NotRequired[str]
+# class AirQuality(TypedDict): # moved before 31DA
+#    air_quality: float | None
+#    air_quality_basis: NotRequired[str]
```

Comment:
I will move them all, I think, in a PR - move may have a tricky rebase, but I think is the best way.

Reply:
Generally, do not leave in commented-out code.

(I do it, but it is a bad habit)

Reply:
Saw the PR, thanks. Will sort out the merge. Commented code was only left for now to mark so we could discuss it.